{"uuid": "b1940322-265d-422c-86ef-ab60a4ec3ea6", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "085ffd40-9943-4142-b98c-b0bcd8b20a46", "event": "runner_on_unreachable", "pid": 97360, "created": "2022-05-01T08:28:13.767440", "parent_uuid": "9e73ed77-7fb2-7934-9437-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "0ebd9885-4a12-4e98-87d4-f3419db12902", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-7934-9437-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-7934-9437-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-05-01T08:28:13.746627", "end": "2022-05-01T08:28:13.767366", "duration": 0.020739, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "b1940322-265d-422c-86ef-ab60a4ec3ea6"}}